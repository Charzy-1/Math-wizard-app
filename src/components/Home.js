import React from 'react';

const Home = () => {
  return (
    <div>
      <h2>Home</h2>
      <p>Mathematics is not all about numbers, equations, computations, or algorithms: It is about understanding - Williams Paul Thurston</p>
    </div>
  );
};

export default Home;
