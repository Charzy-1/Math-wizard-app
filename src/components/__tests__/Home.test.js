import React from 'react';
import { render } from '@testing-library/react';
import Home from '../Home';

// Jest snapshot test
test('Home component matches the snapshot', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});
