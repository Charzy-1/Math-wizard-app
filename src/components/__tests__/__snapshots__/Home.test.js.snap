// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="Home-container"
  >
    <h2>
      Welcome to our Page!
    </h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque sit amet accumsan risus. Vivamus faucibus euismod tortor, sit amet sollicitudin erat. Integer tincidunt eros at sapien vehicula, nec volutpat odio viverra. In dapibus, est sed pharetra suscipit, sapien justo ornare nisi, at fermentum eros ligula ut velit. Fusce quis erat non leo luctus laoreet non id ligula.
    </p>
     
    <br />
    <p>
      Proin malesuada nisl eget libero tincidunt, vitae aliquam massa ullamcorper. Nullam vehicula, erat vel fermentum elementum, turpis felis luctus dui, nec varius erat risus nec metus. Sed id dictum eros, sed dapibus dolor. Nulla facilisi. Phasellus id purus non ligula facilisis blandit. Duis aliquam metus ut ante interdum, id pretium nulla ultricies.
    </p>
  </div>
</DocumentFragment>
`;
